src/components/input-field/input-field.tsx

```diff
@@ -512,8 +513,8 @@ export class InputField {
                 props.href = `tel:${this.value}`;
                 break;
             default:
-                props.href = `${this.value}`;
-                props.target = '_blank';
+                props.href = `${limeLinkHelperService.getHref(this.value)}`;
```

Comment:
Is it obligatory to make it as a string here?

Reply:
Nope, you are right, I will fix it 🙂